src/model/other.model.ts

```diff
@@ -0,0 +1,23 @@
+import { Address, InputDate, natureOfControl, yesNoResponse } from "./data.types.model";
```

Comment:
the name of 'other.ts' is out of sync with naming convention, use the controller/page name ie. beneficial.owner.other.model.ts